website/public/js/controllers/groupsCtrl.js

```diff
@@ -220,6 +220,13 @@ habitrpg.controller("GroupsCtrl", ['$scope', '$rootScope', 'Shared', 'Groups', '
       $scope.usernames = [];
     }
 
+    $scope.filterUser = function(userItem) {
```

Comment:
I still don't like the word 'item', is there some significance about it that I don't understand? Looks like the data it is pulling from is the messages in the group, so would `message` or `msg` be appropriate here?